opengapps-packages.mk

```diff
@@ -34,6 +33,18 @@ GAPPS_PRODUCT_PACKAGES += \
     Turbo
 endif
 
+# api26hack
+ifneq ($(filter $(call get-allowed-api-levels),26),)
```

Comment:
this `filter` statement is consistent with the other uses inside `opengapps-packages.mk`, but after looking at all of them, I think we have the parameters swapped.

the gnu make docs say `$(filter pattern ...,text )`.  Multiple patterns can be given.

Here, `$(filter 26,$(call get-allowed-api-levels))` makes more sense to me.  Note that `get-allowed-api-levels` returns a list of integers 1..n.

This type of refactor could be done in a separate PR.

Reply:
And just to be clear, that separate PR could be done by someone else (e.g. me or anyone else who feels inclined) :-)

Reply:
@jamuir why not use GAPPS_LOCAL_OVERRIDES_PACKAGES := GoogleLoginService ?

Reply:
@alviteri :  I think I understand what you are suggesting.

Add `GAPPS_LOCAL_OVERRIDES_PACKAGES := GoogleLoginService` to the build module for `AndroidPlatformServices`; then the only change needed in `opengapps-packages.mk` is something like this:
```make
if (api level >= 26)
if (architecture is arm64)
  GAPPS_PRODUCT_PACKAGES += \
    AndroidPlatformServices
endif
endif
```

This seems a bit simpler.  I don't know of a good reason not to do it that way, but it's not my PR :-)

Reply:
I agree with using `GAPPS_LOCAL_OVERRIDE_PACKAGES` and (in a follow up commit) fix the filtering.

Btw @alviteri and @jamuir - Thanks for your numerous pull requests. I've added you as collaborators with write access, should you choose to accept it :)

Reply:
thank you @Blystad